README.md

```diff
@@ -38,7 +38,17 @@ We assume you have the following installed:
   * Erlang
   * Elixir
   * Node
-  * PostgreSQL
+
+#### PostgreSQL
```

Comment:
We should be very clear on what is the "recommend" way to get started (which would include using `heroku local`) vs "options". This would be best illustrated with the Postgres install:

---

#### PostgreSQL

##### Recommended

Setup PostgreSQL using a [Docker](https://www.docker.com) container with [docker-compose](./docker-compose.yml):

```shell
> docker-compose up -d
```

##### Optional

Install PostgreSQL directly on your development machine using your operating system's package manager.

---